libiocage/lib/events.py

```diff
@@ -115,51 +117,51 @@ def pending(self, state):
         if new_state is True:
             if self._started_at is not None:
                 raise libiocage.lib.errors.EventAlreadyFinished(event=self)
-            self._started_at = timer()
+            self._started_at = float(timer())
         if new_state is False:
-            self._stopped_at = timer()
+            self._stopped_at = float(timer())
 
         self._pending = new_state
         IocageEvent.PENDING_COUNT += 1 if (state is True) else -1
 
     @property
-    def duration(self):
+    def duration(self) -> typing.Optional[float]:
```

Comment:
why is this float? doesn't time come in better types?

Reply:
this is the delta of two timestamps. what other than float do you suggest to use?